client/my-sites/email/form/mailboxes/types.ts

```diff
@@ -70,7 +69,7 @@ abstract class MailboxFormFieldBase< T > implements MailboxFormField< T > {
 
 class DataMailboxFormField extends MailboxFormFieldBase< string > {
 	isVisible = false;
-	value = uuid_v4();
+	value = crypto.randomUUID() as string;
```

Comment:
Bummer that we need to cast to string. `crypto.randomUUID()` will always return a string after all. Are you sure this is really necessary?

Reply:
It doesn't return a normal string, it returns a specific string like `${string}-${string}-${string}-${string}-${string}`, and below in the file something tries to do `value = normalString` and TS complains because `normalString` doesn't match `${string}-${string}-${string}-${string}-${string}`. That's why I cast without remorse 😄 .